doc/content/verification_and_validation/index.md

```diff
@@ -12,6 +12,7 @@ TMAP8 also contains [example cases](examples/tmap_index.md), which showcase how
 | ver-1b  | [Diffusion Problem with Constant Source Boundary Condition](ver-1b.md)                            |
 | ver-1c  | [Diffusion Problem with Partially Preloaded Slab](ver-1c.md)                                      |
 | ver-1d  | [Permeation Problem with Trapping](ver-1d.md)                                                     |
+| ver-1dc | [Permeation Problem with multiple Trapping](ver-1dc.md)                                           |
```

Comment:
I think we need to think about how to present ver-1d, ver-1da, da, dc, dd in a consistent way. 
ver-1da and ver-1db are currently merged in ver-1d, but maybe it should be separated. 

Not something for you to do now, but something to keep in mind for later.